source/overview.rst

```diff
@@ -21,7 +21,7 @@ at 1 kHz with 5 different interfaces:
 At the same time, you get access to 1 kHz measurements of:
 
  * Measured joint data, such as the position, velocity and link side torque sensor signals.
- * Estimation of externally applied torques and wrenches.
```

Comment:
I was not familiar with the word "wrench" as used in this context, but as far as I can find by searching online, externally applied wrenches contain both torques and forces. This seems to be confirmed by https://github.com/frankaemika/franka_ros/blob/kinetic-devel/franka_control/src/franka_state_controller.cpp#L449-L454